webpack/envs/clientNovoDevelopmentConfig.js

```diff
@@ -43,8 +40,8 @@ export const clientNovoDevelopmentConfig = {
         ],
       },
       {
-        test: /\.(jade|pug)$/,
         include: path.resolve(basePath, "src"),
+        test: /\.(jade|pug)$/,
```

Comment:
Do we still need jade in the novo client?

Reply:
Probably not, these weren't added in this PR and I have a task for cleaning up Webpack once it's stabilized. There is likely a lot that can be removed and deduplicated.